src/applications/vaos/newAppointmentFlow.js

```diff
@@ -115,7 +116,7 @@ export default {
   },
   ccPreferences: {
     url: '/new-appointment/community-care-preferences',
-    next: 'reasonForAppointment',
+    next: 'contactInfo',
```

Comment:
According to [flow chart](https://www.figma.com/file/KGChcQHMrTReo7T7cML418/VAOS-Flow?node-id=246%3A38), if CC, they should not be presented with "reason for appointment" page

Reply:
I don't think this is correct, but not positive.